src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+
+public class Formatter {
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
```

Comment:
Duplicate member. Still present in the TextUi class as well. Which class should contain this data member ?